graphdatascience/query_runner/gds_arrow_client.py

```diff
@@ -77,25 +94,508 @@ def __init__(
+
+        if concurrency:
+            config["concurrency"] = concurrency
+        if undirected_relationship_types:
+            config["undirected_relationship_types"] = undirected_relationship_types
+        if inverse_indexed_relationship_types:
+            config["inverse_indexed_relationship_types"] = inverse_indexed_relationship_types
+
+        self._send_action("CREATE_GRAPH", config)
+
+    def create_graph_from_triplets(
+        self,
+        graph_name: str,
+        database: str,
+        undirected_relationship_types: Optional[List[str]] = None,
+        inverse_indexed_relationship_types: Optional[List[str]] = None,
+        concurrency: Optional[int] = None,
+    ) -> None:
+        """
+        Starts a new graph import process on the GDS server.
+
+        The import process accepts triplets as input data.
+
+        Parameters
+        ----------
+        graph_name : str
+            The name used to identify the graph in the catalog and the import process
+        database: str
+            The name of the database from which the graph will be accessible
+        undirected_relationship_types : Optional[List[str]]
+            A list of relationship types that should be treated as undirected
+        inverse_indexed_relationship_types : Optional[List[str]]
+            A list of relationship types that should be indexed in reverse direction as well
+        concurrency : Optional[int]
+            The number of threads used on the server side when importing the graph
+        """
+
+        config: Dict[str, Any] = {
+            "name": graph_name,
+            "database_name": database,
+        }
+
+        if concurrency:
+            config["concurrency"] = concurrency
+        if undirected_relationship_types:
+            config["undirected_relationship_types"] = undirected_relationship_types
+        if inverse_indexed_relationship_types:
+            config["inverse_indexed_relationship_types"] = inverse_indexed_relationship_types
+
+        self._send_action("CREATE_GRAPH_FROM_TRIPLETS", config)
+
+    def create_database(
+        self,
+        database: str,
+        id_type: Optional[str] = None,
+        id_property: Optional[str] = None,
+        db_format: Optional[str] = None,
+        concurrency: Optional[int] = None,
+        force: bool = False,
+        high_io: bool = False,
+        use_bad_collector: bool = False,
+    ) -> None:
+        """
+        Starts a new graph import process on the GDS server.
+
+        The import process accepts triplets as input data.
+
+        Parameters
+        ----------
+        database: str
+            The name used to identify the database and the import process
+        id_type : Optional[str]
+            Sets the node id type used in the input data. Can be either `INTEGER` or `STRING` (default is `INTEGER`)
+        id_property : Optional[str]
+            The node property key which stores the node id of the input data (default is `originalId`)
```

Comment:
Where are these default values set?

Reply:
In GDS, the values are optional in the config and have defaults on the GDS side.